example/src/index.jsx

```diff
@@ -24,29 +24,29 @@ class App extends React.Component {
             logo={Logo}
             logoDestination="https://edx.org"
             logoAltText="edX"
-            mainMenu={[
```

Comment:
I think I'd leave this in / not commented out, just because it's a more complete example that way.

Reply:
Ops, forgot to uncomment, thank you!